### PR TITLE
V2 calls list

### PR DESCRIPTION
**Pull request by svenlombaert**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -197,6 +197,46 @@ Get a list of all tags
...
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + _items (array)
+            + (object)
+                + id: `247` (number)
+                + description: Recruitment call with Hooli (string)
+                + scheduled_at: `2017-01-12T14:00:00+01:00` (string) - Datetime in ISO 8601 format: yyyy-mm-ddThh:mm:ss+|-hh:mm
+                + duration_timetracked: 120 (number)
+                + done: false (boolean)
```
Comment (jenssegers) [inline apiary.apib]: Q: Can a meeting have more than 2 states being done and not done?

File: apiary.apib [hunk 2]
```diff
@@ -197,6 +197,46 @@ Get a list of all tags
...
+                + employee_id: 1 (number)
```
Comment (jenssegers) [inline apiary.apib]: Embed employee here.

File: apiary.apib [hunk 3]
```diff
@@ -197,6 +197,46 @@ Get a list of all tags
...
+                + customer (object) - A call can be linked to a company or a contact
+                    + id: 174 (number)
+                    + name: Hooli (string)
+                    + _type: 'company' (enum[string])
+                        + Members
+                            + contact
+                            + company
+                + optional_contact (object) - When a call is linked to a company, it could be linked to a contact as well (optional)
```
Comment (jenssegers) [inline apiary.apib]: PP: Use `contact`. For me the `optional_` prefix is not necessary here.

File: apiary.apib (hunk 1 above)
Comment (svenlombaert) [inline apiary.apib]: Nope, not at the moment

File: apiary.apib (hunk 3 above)
Comment (svenlombaert) [inline apiary.apib]: It's to avoid confusion with the customer being linked, which could be a contact

File: apiary.apib (hunk 2 above)
Comment (svenlombaert) [inline apiary.apib]: EmployeeId can go even

**Pull request closed without merging**